clarity/src/vm/docs/mod.rs

```diff
@@ -568,11 +572,81 @@ const XOR_API: SimpleFunctionAPI = SimpleFunctionAPI {
 ",
 };
 
+const BITWISE_XOR_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    snippet: "^ ${1:expr-1} ${2:expr-2}",
+    signature: "(^ i1 i2)",
+    description: "Alias for the `xor` function.  Returns the result of bitwise exclusive or'ing `i1` with `i2`.",
+    example: "(^ 1 2) ;; Returns 3
+(^ 120 280) ;; Returns 352
+",
+};
+
+const BITWISE_AND_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: Some("& (bitwise and)"),
+    snippet: "& ${1:expr-1} ${2:expr-2}",
+    signature: "(& i1 i2)",
+    description: "Returns the result of bitwise and'ing `i1` with `i2`.",
+    example: "(& 24 16) ;; Returns 16
+"
+};
+
+const BITWISE_OR_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: Some("| (bitwise or)"),
+    snippet: "| ${1:expr-1} ${2:expr-2}",
+    signature: "(| i1 i2)",
+    description: "Returns the result of bitwise inclusive or'ing `i1` with `i2`.",
+    example: "(| 4 8) ;; Returns 12
+"
+};
+
+const BITWISE_NOT_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: Some("~ (bitwise not)"),
+    snippet: "~ ${1:expr-1}",
+    signature: "(~ i1)",
+    description: "Returns the result of bitwise not, effectively reversing the bits of `i1`.",
+    example: "(~ 3) ;; Returns -4
+"
+};
+
+const BITWISE_LEFT_SHIFT_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: Some("<< (binary left shift)"),
+    snippet: "<< ${1:expr-1} ${2:expr-2}",
+    signature: "(<< i1 u2)",
+    description: "Moves all the bits in `i1` to the left by the number of places specified in `u2`. 
+New bits are filled with zeros. Shifting a value left by one position is equivalent to multiplying it by 2, 
+shifting two positions is equivalent to multiplying by 4, and so on. 
+
+Observe that the second parameter (number of positions to shift) must be of type uint.
+",
+    example: "(<< 2 u1) ;; Returns 4
+(<< 16 u2) ;; Returns 64
```

Comment:
Can you provide examples of negative integers and unsigned integers?

Reply:
Done!